components/posts/Interactions.js

```diff
@@ -1,27 +1,22 @@
 
 import React, { useState, useEffect } from 'react';
-import { View, StyleSheet } from 'react-native';
+import { View, StyleSheet, Dimensions } from 'react-native';
+import Collapsible from 'react-native-collapsible';
+import Comments from './Comments';
 import IconToggle from '../IconToggle';
 import LikeButton from './LikeButton';
 import PlaybackMenu from './PlaybackMenu';
 import like from '../../utils/like';
 import unlike from '../../utils/unlike';
 
-// Space those icons nicely!
-const IconStyles = StyleSheet.create({
-  'container': {
-    display: 'flex',
-    flexDirection: 'row',
-    justifyContent: 'space-evenly',
-    alignItems: 'center',
-    margin: 10
-  },
-});
+const window = Dimensions.get('window');
+const windowWidth = window.width;
```

Comment:
I noticed that you created a separate Dimensions file so you could probably get rid of this here, and check and make sure that this isn't being used anywhere else also.